chat-commands.js

```diff
@@ -3300,6 +3300,7 @@ exports.commands = {
 		}
 		if (!target) return this.errorReply("Provide a valid format.");
 		let originalFormat = Tools.getFormat(target);
+		// Client handles sending a default format if none is selected.
```

Comment:
Orrr maybe even remove the client's default, so the default can be tripped here...